lib/app_profiler.rb

```diff
@@ -29,11 +32,16 @@ module Viewer
     autoload :SpeedscopeRemoteViewer, "app_profiler/viewer/speedscope_remote_viewer"
   end
 
+  module Backend
+    autoload :Stackprof, "app_profiler/backend/stackprof"
```

Comment:
```suggestion
    autoload :StackprofBackend, "app_profiler/backend/stackprof_backend"
```
I think this will help with not having to qualify `::Stackprof` inside this module, and also disambiguating from the stackprof constant. 

We should follow the same pattern for vernier. Your test files already seem to be named like this.